src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -24,13 +26,21 @@ function CardSection() {
 
     const cardMonth = useMemo(() => DateUtils.getMonthNames(preferredLocale)[(defaultCard?.accountData?.cardMonth ?? 1) - 1], [defaultCard?.accountData?.cardMonth, preferredLocale]);
 
+    let BillingBanner: React.ReactNode | undefined;
+    if (CardSectionUtils.shouldShowPreTrialBillingBanner()) {
+        BillingBanner = <PreTrialBillingBanner />;
+    } else {
+        // TODO: Add other billing banners here.
+    }
```

Comment:
Let's only add the necessary code for now. We can add the conditional once we do work on adding more banners
```suggestion
    const BillingBanner: <PreTrialBillingBanner />;
```

Reply:
@luacmartins Do you mind if we leave the way it is for now? It's because we are going to add more billing banners very soon (in the next days)

Reply:
As a principle we are discouraged to add any code that goes beyond the purpose of this PR. 
So, I think we can integrate the suggested changes and add code for further banners as and when we take it up.

Reply:
Adjusted!